module-system/sov-state/src/prover_storage.rs

```diff
@@ -16,27 +17,28 @@ use crate::{MerkleProofSpec, Storage};
 
 pub struct ProverStorage<S: MerkleProofSpec> {
     db: StateDB,
+    native_db: NativeDB,
```

Comment:
Now we have two `DBs`. Is it possible to guarantee atomic update between the two? If not then let's use the same DB with different column families and `atomic flush` from rocks db?

Reply:
Tracked in https://github.com/Sovereign-Labs/sovereign-sdk/issues/791!